pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>io.annot8</groupId>
   <artifactId>annot8-api</artifactId>
-  <version>1.1</version>
+  <version>1.2</version>
```

Comment:
1.2-SNAPSHOT?

Reply:
I didn't mark it as SNAPSHOT as it wasn't 1.1-SNAPSHOT and I assume we are not publishing SNAPSHOTs anywhere so make them of limited use. But I'm not against it.